<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Preferences</class>
 <widget class="QDialog" name="Preferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>preferences_preferences</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_1">
      <attribute name="title">
       <string>preferences_basic</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>12</number>
       </property>
       <property name="leftMargin">
        <number>12</number>
       </property>
       <property name="topMargin">
        <number>12</number>
       </property>
       <property name="rightMargin">
        <number>12</number>
       </property>
       <property name="bottomMargin">
        <number>12</number>
       </property>
       <item>
        <widget class="QComboBox" name="lang">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QComboBox::AdjustToMinimumContentsLengthWithIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="theme"/>
       </item>
       <item>
        <widget class="QComboBox" name="video_driver"/>
       </item>
       <item>
        <widget class="QCheckBox" name="showPlayButtons">
         <property name="text">
          <string>preferences_show_play_buttons_on_cards_with</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="interrupt_audio">
         <property name="text">
          <string>preferences_interrupt_current_audio_when_answering</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="pastePNG">
         <property name="text">
          <string>preferences_paste_clipboard_images_as_png</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="paste_strips_formatting">
         <property name="text">
          <string>preferences_paste_without_shift_key_strips_formatting</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="ignore_accents_in_search">
         <property name="text">
          <string>preferences_ignore_accents_in_search</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="new_import_export">
         <property name="text">
          <string notr="true">New import/export handling (beta)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="useCurrent">
         <item>
          <property name="text">
           <string>preferences_when_adding_default_to_current_deck</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>preferences_change_deck_depending_on_note_type</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="search_text_label">
         <property name="text">
          <string>preferences_default_search_text</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="default_search_text">
         <property name="placeholderText">
          <string>preferences_default_search_text_example</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>preferences_user_interface_size</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="uiScale">
           <property name="suffix">
            <string notr="true">%</string>
           </property>
           <property name="minimum">
            <number>100</number>
           </property>
           <property name="maximum">
            <number>200</number>
           </property>
           <property name="singleStep">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>preferences_scheduling</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QCheckBox" name="showEstimates">
         <property name="text">
          <string>preferences_show_next_review_time_above_answer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="showProgress">
         <property name="text">
          <string>preferences_show_remaining_card_count_during_review</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="dayLearnFirst">
         <property name="text">
          <string>preferences_show_learning_cards_with_larger_steps</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="legacy_timezone">
         <property name="text">
          <string>preferences_legacy_timezone_handling</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="sched2021">
         <property name="toolTip">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>preferences_v3_scheduler</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="newSpread"/>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_4">
         <property name="spacing">
          <number>12</number>
         </property>
         <item row="1" column="2">
          <widget class="QLabel" name="label_29">
           <property name="text">
            <string>preferences_mins</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="timeLimit">
           <property name="maximum">
            <number>9999</number>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="dayOffset">
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="maximum">
            <number>23</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_30">
           <property name="text">
            <string>preferences_timebox_time_limit</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_40">
           <property name="text">
            <string>preferences_hours_past_midnight</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_24">
           <property name="text">
            <string>preferences_learn_ahead_limit</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="label_39">
           <property name="text">
            <string>preferences_mins</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="lrnCutoff">
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="maximum">
            <number>999</number>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_23">
           <property name="text">
            <string>preferences_next_day_starts_at</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>16</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>preferences_network</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>12</number>
       </property>
       <property name="leftMargin">
        <number>12</number>
       </property>
       <property name="topMargin">
        <number>12</number>
       </property>
       <property name="rightMargin">
        <number>12</number>
       </property>
       <property name="bottomMargin">
        <number>12</number>
       </property>
       <item>
        <layout class="QHBoxLayout">
         <property name="spacing">
          <number>10</number>
         </property>
         <item>
          <layout class="QVBoxLayout">
           <item>
            <widget class="QLabel" name="syncLabel">
             <property name="text">
              <string>preferences_synchronisation</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
             <property name="openExternalLinks">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="syncMedia">
             <property name="text">
              <string>preferences_synchronize_audio_and_images_too</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="syncOnProgramOpen">
             <property name="text">
              <string>preferences_automatically_sync_on_profile_openclose</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="autoSyncMedia">
             <property name="text">
              <string>preferences_periodically_sync_media</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="fullSync">
             <property name="text">
              <string>preferences_on_next_sync_force_changes_in</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QPushButton" name="syncDeauth">
               <property name="text">
                <string notr="true">LOGOUT</string>
               </property>
               <property name="autoDefault">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="syncUser">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>1</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QPushButton" name="media_log">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>preferences_backups</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>12</number>
       </property>
       <property name="leftMargin">
        <number>12</number>
       </property>
       <property name="topMargin">
        <number>12</number>
       </property>
       <property name="rightMargin">
        <number>12</number>
       </property>
       <property name="bottomMargin">
        <number>12</number>
       </property>
       <item>
        <widget class="QLabel" name="backup_explanation">
         <property name="text">
          <string>preferences_backup_explanation</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="4" column="1">
          <widget class="QSpinBox" name="monthly_backups">
           <property name="maximum">
            <number>9999</number>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="daily_backups">
           <property name="maximum">
            <number>9999</number>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>preferences_daily_backups</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>preferences_monthly_backups</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QSpinBox" name="weekly_backups">
           <property name="maximum">
            <number>9999</number>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>preferences_weekly_backups</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>preferences_minutes_between_backups</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="minutes_between_backups">
           <property name="minimum">
            <number>5</number>
           </property>
           <property name="maximum">
            <number>9999</number>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <spacer name="horizontalSpacer_8">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="openBackupFolder">
         <property name="text">
          <string>preferences_you_can_restore_backups_via_fileswitch</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>preferences_note_media_is_not_backed_up</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>59</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_21">
     <property name="text">
      <string>preferences_some_settings_will_take_effect_after</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close|QDialogButtonBox::Help</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lang</tabstop>
  <tabstop>theme</tabstop>
  <tabstop>video_driver</tabstop>
  <tabstop>showPlayButtons</tabstop>
  <tabstop>interrupt_audio</tabstop>
  <tabstop>pastePNG</tabstop>
  <tabstop>paste_strips_formatting</tabstop>
  <tabstop>ignore_accents_in_search</tabstop>
  <tabstop>new_import_export</tabstop>
  <tabstop>useCurrent</tabstop>
  <tabstop>default_search_text</tabstop>
  <tabstop>uiScale</tabstop>
  <tabstop>showEstimates</tabstop>
  <tabstop>showProgress</tabstop>
  <tabstop>dayLearnFirst</tabstop>
  <tabstop>legacy_timezone</tabstop>
  <tabstop>sched2021</tabstop>
  <tabstop>newSpread</tabstop>
  <tabstop>dayOffset</tabstop>
  <tabstop>lrnCutoff</tabstop>
  <tabstop>timeLimit</tabstop>
  <tabstop>syncMedia</tabstop>
  <tabstop>syncOnProgramOpen</tabstop>
  <tabstop>autoSyncMedia</tabstop>
  <tabstop>fullSync</tabstop>
  <tabstop>syncDeauth</tabstop>
  <tabstop>media_log</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>minutes_between_backups</tabstop>
  <tabstop>daily_backups</tabstop>
  <tabstop>weekly_backups</tabstop>
  <tabstop>monthly_backups</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Preferences</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>285</x>
     <y>439</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Preferences</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>332</x>
     <y>439</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
